content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4027,6 +4028,114 @@ Informs that a chatting Customer's session fields were updated. The webhook will
 
 </Section>
 
+## Configuration
+
+<Section>
+
+<Text>
+
+### `group_created`
+
+Informs that group has been created on license.
```

Comment:
`group` -> `a group`
I'd also replace `license` with `a license` or `the license`, but not 100% sure - ask Ola :)

Reply:
I try to avoid "on the license" in favor of "within a license".